src/middleware/node/middleware.ts

```diff
@@ -77,11 +77,16 @@ export async function middleware(
     return true;
   }
 
-  const eventName = request.headers["x-github-event"] as WebhookEventName;
-  const signatureSHA256 = request.headers["x-hub-signature-256"] as string;
-  const id = request.headers["x-github-delivery"] as string;
-
-  options.log.debug(`${eventName} event received (id: ${id})`);
+  const {
+    "x-github-event": name,
+    "x-hub-signature-256": signature,
+    "x-github-delivery": id,
+  } = request.headers as {
+    "x-github-event": WebhookEventName;
+    "x-hub-signature-256": string;
+    "x-github-delivery": string;
+  };
```

Comment:
Why do it this way, it's much more clear and concise the way it was before. The variable names were more descriptive

Reply:
```suggestion
  const eventName = request.headers["x-github-event"] as WebhookEventName;
  const signatureSHA256 = request.headers["x-hub-signature-256"] as string;
  const id = request.headers["x-github-delivery"] as string;
```